README.md

```diff
@@ -133,23 +133,31 @@ query Terms {
         alternateName
       }
     }
-    terms {
-      uri
-      prefLabel
-      altLabel
-      hiddenLabel
-      scopeNote
-      broader {
-        uri
-        prefLabel
-      }
-      narrower {
-        uri
-        prefLabel
+    result {
```

Comment:
Thinking out load: is it useful to make the `source` object also a part of the `result` structure? With our current implementation it's unlikely that errors will occur when loading the `source` information, but would it make the API design more forward-compatible?

Reply:
The errors I added here I scoped per source, so I’d prefer to keep the source data (always available because it comes from our catalog) out of the variable bit (success with terms or failure with error message).

I case of total breakage (which obviously should occur as little as possible 😅), we can still use the GraphQL `errors` property on the global level.

Reply:
👍🏻